about.html

```diff
@@ -0,0 +1,135 @@
+                            GLOBAL SUMMIT 2015" class="title_image">
+                        <p class="about">Welcome to the International Stuttering
+                            Awareness Day (ISAD) Online Conference,
+                            October 1-31, 2022, celebrating the 25th time the
+                            Online Conference has been held!
+                            An enthusiastic team made up of Anita Blom, Hanan
+                            Hurwitz, McKenzie Jemmett, Pamela Mertz,
+                            Tiffani Kittilstved, Jaime Michise, Elias Apreko,
+                            Shilpa Sagwal and Bruce Imhoff will coordinate
+                            this year’s event, continuing the hard work of Judy
+                            Kuster who ran previous Online Conferences.
+                            The ISAD online conference is a private initiative
+                            by volunteers, and totally independent from any
+                            organisation.This year, the Online Conference will
+                            start October 1 and finish October 31,
+                            acknowledging the many events around the world on
+                            October 22, International Stuttering Awareness Day.
+                            If browsing the conference on a computer or tablet
+                            device, navigate through the conference using the
+                            menu bar at the top. If using a mobile device, use
+                            the ‘Menu’ button at the top.
+                        </p>
+                        <span>
+                            <p class="contactus">Please contact us via email for
+                                any questions about LIVE AWARE summit 2022</p>
+                            <a class="mail" href="/">liveaware2022@cckorea.org ></a>
+                        </span>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <footer>
+            <div class="cover-white cover-darker-desktop">
+                <div class="container">
+                    <div>
+                        <a href="/">
+                            <img src="images/logo_cck_w_208x35.png" alt="logo">
+                        </a>
+                        <div>
+                            <span><b>2015 Creative Commons Korea. Some Rights Reserved.</b></span>
+                            <p>Lorem ipsum dolor sit amet, consectetur
+                                adipisicing elit.
+                                Eaque sed in perferendis, quod aperiam optio
+                                maiores omnis ipsum,
+                                et pariatur tempore fuga quisquam reprehenderit
+                                veritatis.</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
```

Comment:
- Please include the section attached in the picture referenced below in your design. I noticed you removed it (due to a misunderstanding in my last review) 🙏 🙏 
- 
<img width="553" alt="past conf behance" src="https://user-images.githubusercontent.com/73607512/198550954-fc9660c5-5c18-46b9-a2bb-34ecf440f0e7.PNG">

**OPTIONAL** - In my last review I asked that you removed the sponsor section, I should have mentioned that it should be removed only in the desktop version only. It would be nice if you could implement this 😄